boards/arm/s32z270dc2_r52/s32z270dc2_r52.dtsi

```diff
@@ -89,3 +89,15 @@
 &swt4 {
 	clock-frequency = <48000000>;
 };
+
+&can0 {
+	pinctrl-0 = <&can0_default>;
+	pinctrl-names = "default";
+	clock-frequency = <80000000>;
+	bus-speed = <1000000>;
+	sample-point = <875>;
+	sjw = <2>;
```

Comment:
Can this be changed to the Zephyr "default" of 1?

Reply:
yes, can